rust/main/chains/hyperlane-sealevel/src/rpc/client.rs

```diff
@@ -199,16 +216,74 @@ impl SealevelRpcClient {
             .map_err(ChainCommunicationError::from_other)
     }
 
-    pub async fn send_and_confirm_transaction(
+    pub async fn send_transaction(
         &self,
         transaction: &Transaction,
+        skip_preflight: bool,
```

Comment:
when would we set this to `false`?

Reply:
hmm good point - despite us not setting it to false atm I think it's a little nice to have in the interface as a way to indicate that we're doing something a little non-standard. But don't feel strongly. Gonna leave as is and we can change later if we want